applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_analysis.py

```diff
@@ -95,7 +95,7 @@ def _CreateProcesses(self, parameter_name, initialization_order):
                 info_msg += "Refer to \"https://github.com/KratosMultiphysics/Kratos/wiki/Common-"
                 info_msg += "Python-Interface-of-Applications-for-Users#analysisstage-usage\" "
                 info_msg += "for a description of the new format"
-                KratosMultiphysics.Logger.PrintInfo("StructuralMechanicsAnalysis", info_msg)
+                KratosMultiphysics.Logger.PrintWarning("StructuralMechanicsAnalysis", info_msg)
```

Comment:
@philbucher The only thing I changed is this, this way the people will start to migrate their tests due to the verbosity

Reply:
good idea :+1: